lms/static/scripts/frontend_apps/components/test/AuthorizationModal-test.js

```diff
@@ -21,10 +21,9 @@ describe('AuthorizationModal', () => {
 
   it('provides an authorization button', () => {
     const wrapper = createComponent();
-    const authorizeButton = wrapper.find('LabeledButton');
```

Comment:
This test for another component (`AuthorizationModal`) failed from changes to `AuthButton`. I updated it to test for the presence of `AuthButton` (ensuring this component actually renders an `AuthButton`) but eliminated the bits that involved implementation details of `AuthButton` (violating unit boundaries).